mne/epochs.py

```diff
@@ -1233,6 +1242,16 @@ def __getitem__(self, key):
             key = [key]
 
         if isinstance(key, (list, tuple)) and isinstance(key[0], string_types):
+            if any(["/" in k_i for k_i in epochs.event_id.keys()]):
```

Comment:
Compare: it's the same object that's being referenced in all of the unchanged lines below.